.github/workflows/interpreter.yml

```
@@ -0,0 +1,64 @@
+name: Interpreter Test
```

Naming suggestion to keep consistency with the rest of the workflows.

```suggestion
name: Interpreter CI
```